app/src/main/res/values/strings.xml

```diff
@@ -9,5 +9,11 @@
         <item>Organizer</item>
         <item>Admin</item>
     </string-array>
+    <string-array name="entrant_status">
```

Comment:
i think we might also need to include a pending (when organizer hasn't made decision yet), lost (entrant loses initial lottery), and declined status (entrant doesn't want to go after being accepted). also just curious, what do the waitlist and invited status mean?

Reply:
I guess it's kinda vague but those statuses you're describing are what is there. On waitlist means "pending" as there's been no decision yet (Just every entrant who has joined the waitlist) and canceled is the declined status. They canceled their entry. Invited just means that they won the lottery and they've recieved a notification to join the event. Also I don't think in the user stories we need to worry about who lost the lottery